pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
+                        <compatibleSinceVersion>2.9</compatibleSinceVersion>
+                        <dependentWarExcludes>org.apache.jclouds*</dependentWarExcludes>
+                        <archive>
+                            <manifestEntries>
+                                <Mask-Classes>com.google.common</Mask-Classes>
```

Comment:
Not masking guava classes causes loading jenkins' core guava (11.0.1) and a 
`java.lang.NoSuchMethodError: com.google.common.base.Objects$ToStringHelper.omitNullValues()Lcom/google/common/base/Objects$ToStringHelper;
	at org.openstack4j.openstack.compute.domain.NovaServer.toString(NovaServer.java:281)
	at jenkins.plugins.openstack.compute.JCloudsSlaveTemplate.provision(JCloudsSlaveTemplate.java:279)
`